content/basics/ably.textile

```diff
@@ -0,0 +1,100 @@
+
+Read more in our "Push notifications section":/push/.
+
+h2(#infrastructure). About the Infrastructure
+
+The importance of a seamless experience can't be overstated, that's why we've built the Ably infrastructure around the "Four Pillars of Dependability":https://ably.com/four-pillars-of-dependability :
+
+* "Performance":#performance
+* "Integrity":#integrity
+* "Reliability":#reliability
+* "Availability":#availability
+
+h3(#performance). Performance
+
+Performance is more than just minimizing latency and bandwidth requirements; it's also about minimizing the variance in them and providing predictability to developers.
+
+If you know Ably’s median global latencies and bandwidth will always be within specific operating boundaries it provides a level of certainty in uncertain operating conditions. You can design, build, and scale features around this certainty, confident they’ll perform as expected under various conditions.
+
+h3(#integrity). Integrity
+
+When apps rely on a sequence of messages that mutually depend upon one another, like chat, Ably maintains the end-to-end integrity of them. This simplifies app architecture: there’s no need to handle missed, unordered, or duplicate messages.
+
+This frees you from design limitations so you can focus on solving the challenges that really matter, not the frustrating realtime edge cases you’re otherwise forced to think about and develop around.
+
+Due to our more modern system architecture we’re able to actually provide these guarantees at scale - something other pub/sub messaging providers with similar claims struggle to deliver on.
+
+h3(#reliability). Reliability
+
+It's important to be able to trust your providers and Ably’s platform is fault tolerant to provide you with that assurance. We can continue to operate even if a component, or multiple components, should fail. We’ve designed around statistical risks of failure to ensure sufficient redundancy at regional and global levels to ensure continuity of service even in the face of multiple infrastructure failures.
+
+h3(#availability). Availability
+
+Ably is meticulously designed to be elastic and highly-available, providing the uptime and elastic scale required for stringent and demanding realtime requirements. Under load, achieving availability and elasticity isn’t just traditional mechanics like failover, it’s about managing capacity.
+
+Ably’s mathematically grounded design means we can transparently share operating boundaries we monitor to ensure capacity and therefore availability, helping you understand the type of scale and elasticity capable with Ably.
+
+RELATED-RESOURCES
```

Comment:
Might be worth noting which resources to include whilst you're familiar with the content.